File: 448-find-all-numbers-disappeared-in-an-array/448-find-all-numbers-disappeared-in-an-array.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        s = []
        s = [i for i in range(1, len(nums) + 1)]
        out = set(s) - set(nums)
        return out

File: 448-find-all-numbers-disappeared-in-an-array/448-find-all-numbers-disappeared-in-an-array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        vector<int> out;
        set<int> s;
        
        for (int i = 0; i < nums.size(); i++) 
            s.insert(nums[i]);

        for (int i = 1; i <= nums.size(); i++) {
            if (s.find(i) == s.end()) 
                out.push_back(i);
        }
        return out;
    }
};